public/index.html

```diff
@@ -38,7 +38,7 @@
             <section class="intake">
                 <p>Congue aciliquis atis, faccummy aliqui ing vero commodolor dolesecte, nec eugiamconse conubia.</p>
 
-                <form onsubmit="return false">
+                <form onsubmit="return false;">
```

Comment:
This no-op change is causing a conflict with other changes that have already merged out. Can you remove this semi-colon?